### PR TITLE
Improve the doc comment for MaxAccepts

### PR DESCRIPTION
**Pull request by davidfowl**
- Added defaults
- Explain that it's about concurrent

### CODE REVIEW

File: src/Servers/HttpSys/src/HttpSysOptions.cs [hunk 1]
```diff
@@ -68,8 +68,11 @@ public string RequestQueueName
         public ClientCertificateMethod ClientCertificateMethod { get; set; } = ClientCertificateMethod.AllowCertificate;
 
         /// <summary>
-        /// The maximum number of concurrent accepts.
+        /// The maximum number of concurrent accepts dequeuing requests from the Http.sys queue.
```
Comment (JamesNK) [inline src/Servers/HttpSys/src/HttpSysOptions.cs:71]: As someone who doesn't know what this property does, the comment doesn't help 😄

Comment (davidfowl) [reply src/Servers/HttpSys/src/HttpSysOptions.cs:71]: LOL. It's *very* advanced but I wanted to at least have the defaults there and explain that its about dequeuing...

Comment (davidfowl) [inline src/Servers/HttpSys/src/HttpSysOptions.cs:71]:
```suggestion
        /// The number of concurrent workers draining requests from the Http.sys queue.
```

Comment (davidfowl) [reply src/Servers/HttpSys/src/HttpSysOptions.cs]: Is it better?

File: src/Servers/HttpSys/src/HttpSysOptions.cs [hunk 2]
```diff
@@ -68,8 +68,11 @@ public string RequestQueueName
         public ClientCertificateMethod ClientCertificateMethod { get; set; } = ClientCertificateMethod.AllowCertificate;
 
         /// <summary>
-        /// The maximum number of concurrent accepts.
+        /// The number of concurrent workers draining requests from the Http.sys queue.
```
Comment (pranavkm) [inline src/Servers/HttpSys/src/HttpSysOptions.cs:71]:
```suggestion
        /// Gets or sets the number of concurrent workers draining requests from the Http.sys queue.
```

File: src/Servers/HttpSys/src/HttpSysOptions.cs [hunk 3]
```diff
@@ -68,8 +68,11 @@ public string RequestQueueName
...
         /// </summary>
+        /// <remarks>
+        /// Defaults to 5 times the number of processors.
```
Comment (pranavkm) [inline src/Servers/HttpSys/src/HttpSysOptions.cs:74]:
```suggestion
        /// Defaults to 5 times the number of processors as returned by <see cref="Environment.ProcessorCount" />.
```

**Review by pranavkm — APPROVED** [review]

**Review by pranavkm — CHANGES_REQUESTED** [review]
Bad David.

**Comment by davidfowl** [comment]
Automerge FTW

**Comment by halter73** [comment]
Automerge really does break the approve-pr-pending-nits workflow. As nice as it is for the PR submitter, it means the reviewer has to come back to approve the PR later if they want to leave minor feedback.

**Comment by davidfowl** [comment]
😂

**Pull request merged**